src/main/java/com/jalasoft/jfc/controller/MetadataConverterController.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.controller;
+
+import com.jalasoft.jfc.model.Param;
+import com.jalasoft.jfc.model.exception.ConvertException;
+import com.jalasoft.jfc.model.exception.Md5Exception;
+import com.jalasoft.jfc.model.exception.ZipJfcException;
+import com.jalasoft.jfc.model.metadata.MetadataConverter;
+import com.jalasoft.jfc.model.result.FileResponse;
+import com.jalasoft.jfc.model.result.MessageResponse;
+import com.jalasoft.jfc.model.result.Response;
+import com.jalasoft.jfc.model.utility.FileServiceController;
+import com.jalasoft.jfc.model.utility.FolderRemover;
+import com.jalasoft.jfc.model.utility.LinkGenerator;
+import com.jalasoft.jfc.model.utility.Md5Checksum;
+import com.jalasoft.jfc.model.utility.PathJfc;
+import com.jalasoft.jfc.model.utility.ZipFolder;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Extracts metadata from a uploaded file.
```

Comment:
Review this code line, it would be: "Extracts metadata of an uploaded file"

Reply:
ok thank you for your example